### ISSUE TITLE
Theme missing? Link?

### ISSUE THREAD

**Issue by mpatton125** [jay-johnson/docker-nginx-sphinx-bootstrap#1]
Hi there,

Thanks for your work on this.

One thing I noticed, the _theme directory just has a symbolic link that doesn't go anywhere?

docker-nginx-sphinx-bootstrap/source/_themes/bootstrap

M.

**Comment by jay-johnson** [comment]
Good catch. I noticed the same thing, and it's a carry over from the original repo https://github.com/ryan-roemer/sphinx-bootstrap-theme/tree/master/demo/source/_themes Not sure what it was used for so I decided to leave it alone for the first release.

Just checked it is not even on my ec2 host:

```
[root@ip-10-0-1-63 source]# pwd
/opt/blog/repo/source
[root@ip-10-0-1-63 source]# ls _themes/
[root@ip-10-0-1-63 source]# 
```
